src/features/user/MFV2/ProfileLayout/index.tsx

```diff
@@ -46,7 +46,7 @@ const ProfileLayout = () => {
         )}
       </section>
       <section id="map-container" className={styles.mapContainer}>
-        <ContributionsMap />
+        {contributionsResult?.stats ? <ContributionsMap /> : <ProfileLoader />}
```

Comment:
Perhaps we need to wait for the stats to be available before showing them to the user as "0" values.